modules/codegen/src/smithy4s/codegen/transformers/AwsStandardTypesTransformer.scala

```diff
@@ -96,12 +97,33 @@ class AwsStandardTypesTransformer extends ProjectionTransformer {
         t.isInstanceOf[DefaultTrait] || t.isInstanceOf[BoxTrait]
       })
 
+  private def hasStandardName(shape: Shape): Boolean =
+    AwsStandardTypesTransformer.standardSimpleShapes.contains(
```

Comment:
```suggestion
    AwsStandardTypesTransformer.standardSimpleShapes(
```

I think you could get away by calling apply on the Set directly, but this is minor